runconfig/parse.go

```diff
@@ -513,3 +505,31 @@ func ParseDevice(device string) (DeviceMapping, error) {
 	}
 	return deviceMapping, nil
 }
+
+// ParseVolumesIntoBinds parses any volumes passed by the client and moves any
+// volumes which are in Config.Volumes into HostConfig.Binds.
+func ParseVolumesIntoBinds(c *Config, hc *HostConfig) (*Config, *HostConfig, error) {
```

Comment:
By literal I mean the path is expected to be left alone. The path is the path in the container, nothing else.